--- a/src/CaptiveLogin.vala
+++ b/src/CaptiveLogin.vala
@@ -29,7 +29,7 @@
 
     private const string TITLE = _("Log in");
     private const string DUMMY_URL = "http://elementary.io/capnet-assist";
-    private const string GENERATE_204_URL = "http://connectivitycheck.android.com/generate_204";
+    private const string GENERATE_204_URL = "http://elementary.io/generate_204";
 
     private WebKit.WebView web_view;
     private Gtk.ToggleButton header_button;
